samples/tfm_integration/tfm_psa_test/sample.yaml

```diff
@@ -0,0 +1,24 @@
+common:
+    tags: tfm
+    platform_allow: mps2_an521_nonsecure lpcxpresso55s69_ns nucleo_l552ze_q_ns
```

Comment:
Only these?

Reply:
nrf platforms need an upmerge of trusted-firmware-m (and psa-arch-tests)

Reply:
I tried building for the different platforms and fixed the list now.